output.go

```diff
@@ -91,7 +91,7 @@ func debug(p ...interface{}) {
 		} else {
 			printStr = printer(color.FgMagenta, "DBUG", toPrint)
 		}
-		fmt.Printf(printStr)
+		fmt.Print(printStr)
```

Comment:
As it was `Printf()` before, this is just a sanity check, but it is intentional that it is `Print()` and not `Println()` yeah?

Reply:
Yeah sorry had this change lying around for a while, it is supposed to be Print. Printf was kind of like a format string vulnerability lol, not that those exist in go. The newline is added in toPrint via Sprintln.